com.github.cfcurtis.pdfstitcher.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Version=1.0
+Version=0.5.2
```

Comment:
Version here isn't the software version. It's the desktop file specification version.

it's the metainfo.xml you need to change.